spec/functions/README.md

```diff
@@ -16,13 +15,19 @@
    1. [`:date`](datetime.md#the-date-function)
    1. [`:time`](datetime.md#the-time-function)
 
-## Introduction
-
 This section defines the **_<dfn>default functions</dfn>_**
 which are REQUIRED for conformance with this specification,
 along with _default functions_ that SHOULD be implemented to support
 additional functionality.
 
+> [!IMPORTANT]
+> _Functions_ that are not marked **Draft** are **Stable** and subject to
+> the provisions of the [stability policy](../intro.md#stability-policy).
+>
+> _Functions_ and _options_ marked as **Draft** are not stable.
```

Comment:
After the other change I suggest here, we only have Draft functions.

```suggestion
> _Functions_ marked as **Draft** are not stable.
```

Reply:
That's true **now** but might not be in the future. We shouldn't have to modify the introduction if/when we introduce draft options in the future.

It should be "or" though:

```suggestion
> _Functions_ or _options_ marked as **Draft** are not stable.
```

Reply:
Eh, sure. I still think it'd be easier to just redefine this yet again later, but it's not important.